jenkins/tippecanoe_create_tiles_Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+pipeline {
+  agent none 
```

Comment:
You want to specify the slave agent for Jenkins:

```
agent {
        node {
            label 'node:slave'
        }
    }
```

Ideally we'd like to build nothing on the master node

Reply:
We don't need this right now, but if in the future we needed a heavy-duty node with extra RAM or something, could we request that here?

Reply:
I'd just ping me on Slack for that. We can certainly create one for you

Reply:
Cool